src/cfnlint/rules/resources/HardCodedArnProperties.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.

```

Comment:
```suggestion
Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

[can remove dates now](https://github.com/aws-cloudformation/cfn-python-lint/pull/1528)